saw-remote-api/python/saw_client/crucible.py

```diff
@@ -604,18 +622,21 @@ def to_json(self) -> Any:
 
 # It's tempting to name this `global` to mirror SAWScript's `llvm_global`,
 # but that would clash with the Python keyword `global`.
-def global_var(name: str) -> SetupVal:
+def global_var(name: str) -> GlobalVarVal:
     """Returns a pointer to the named global ``name`` (i.e., a ``GlobalVarVal``)."""
     return GlobalVarVal(name)
 
-# FIXME Is `Any` too permissive here -- can we be a little more precise?
-def cryptol(data : Any) -> 'CryptolTerm':
-    """Constructs a Cryptol value from ``data`` (i.e., a ``CryptolTerm``, which is also a ``SetupVal``).
+def cry(s : str) -> CryptolTerm:
+    """Like ``cry`` from the Cryptol bindings, but retuns a ``CryptolTerm``
+    (which is also a ``SetupVal``)."""
+    return CryptolTerm(s)
 
-    ``data`` should be a string literal representing Cryptol syntax or the result of a Cryptol-realted server computation."""
-    return CryptolTerm(data)
+def cry_f(s : str) -> CryptolTerm:
+    """Like ``cry_f`` from the Cryptol bindings, but retuns a ``CryptolTerm``
```

Comment:
Is it worth duplicating some/all of the Cryptol `cry_f` docs here? I know it means possibly more "maintenance work", but it might be a lot more convenient for users to just get the real docs here instead of a pointer they have to manually chase.